index.js

```diff
@@ -1,23 +1,19 @@
-const loopWhile = require('deasync').loopWhile
-const processor = require('./processor')
 
+const childProcess = require('child_process');
 module.exports = (css, settings) => {
   const cssWithPlaceholders = css
     .replace(/%%styled-jsx-placeholder-(\d+)%%/g, (_, id) =>
       `/*%%styled-jsx-placeholder-${id}%%*/`
     )
-  let processedCss
-  let wait = true
-
-  function resolved(result) {
-    processedCss = result
-    wait = false
-  }
-
-  processor(cssWithPlaceholders, settings)
-    .then(resolved)
-    .catch(resolved)
-  loopWhile(() => wait)
+  let cssWithPlaceholdersArg = cssWithPlaceholders
+  let settingsArg = settings
+  if(cssWithPlaceholdersArg)
+    cssWithPlaceholdersArg = cssWithPlaceholders.replace(/\"/g,`\\"`)
+  if(settingsArg)
+    settingsArg = JSON.stringify(settings).replace(/\"/g,`\\"`)
+  const result = childProcess.execSync(`node ${__dirname}/processor.js "${cssWithPlaceholdersArg}" "${settingsArg}"`,
+            { encoding: 'utf8', maxBuffer: 50 * 1024 * 1024 })
```

Comment:
@tldrd0117 hi thanks a lot for this PR! Probably the best way to pass the CSS and the settings to the child process is the following:

```js
const result = exec(`node ${path.resolve(__dirname, "processor.js")}`, {
  input: JSON.stringify({
    css: cssWithPlaceholders,
    settings
  }),
  encoding: "utf8",
});
```

Which you can read in the child process with:

```js
let input = "";
process.stdin.on("data", (data) => {
  input += data;
});
process.stdin.on("end", () => {
  console.log(JSON.parse(input));
});
```

Reply:
hi thanks for review, i think we need maxbuffer option because default size is 1024 * 1024 and preventing ENOBUFS Error. I'll commit again